riscv/encoding.h

```diff
@@ -954,9 +954,9 @@
 #define MATCH_SINVAL_VMA 0x16000073
 #define MASK_SINVAL_VMA  0xfe007fff
 #define MATCH_SFENCE_W_INVAL 0x18000073
-#define MASK_SFENCE_W_INVAL  0xfff07fff
+#define MASK_SFENCE_W_INVAL  0xffffffff
 #define MATCH_SFENCE_INVAL_IR 0x18100073
-#define MASK_SFENCE_INVAL_IR  0xfff07fff
+#define MASK_SFENCE_INVAL_IR  0xffffffff
```

Comment:
@daniellustig in cb8f09a4d6fed30527fdd832b885c898b4591a5f you added these mask values by hand. For some reason the values here don't match with riscv-opcodes, resulting in this diff when we recreate the encoding.h from riscv-opcodes.

Can you confirm which repo has the correct mask value?

Reply:
`0xffffffff` is correct.  I originally added the values manually into spike because at the time, encoding.h in riscv-isa-sim and encoding.h in riscv-opcodes had diverged.  I then later fixed a bug in the version in riscv-opcodes (https://github.com/riscv/riscv-opcodes/pull/78), but then apparently I never propagated that fix over to spike.  My mistake, sorry.

If you don't mind these two lines of diff being mixed into your unrelated PR, please feel free to just include them into your merge.  Otherwise, I can push a separate fix for those two lines.  Let me know which you prefer.

(I assume there's still not a more automated process of keeping encoding.h in sync across the various repos that have a copy?)

Reply:
Fixed by #848; @mjosaarinen please rebase, regenerating encoding.h from the latest riscv-opcodes, and it should remove all these non-Krypto changes from encoding.h.